justfile

```diff
@@ -15,7 +22,6 @@ build-wasm mode=default_mode:
 	cargo build --package csslsrs --target wasm32-unknown-unknown {{ if mode == "release" {"--release"} else if mode == "benchmark" {"--profile benchmark"} else {""} }} --features wasm
 	wasm-bindgen ./target/wasm32-unknown-unknown/{{mode}}/csslsrs.wasm --out-dir ./packages/csslsrs/src/generated --target=experimental-nodejs-module {{ if mode == "release" { "" } else { "--keep-debug" } }}
 	{{ if mode == "release" { "wasm-opt -O4 ./packages/csslsrs/src/generated/csslsrs_bg.wasm -o ./packages/csslsrs/src/generated/csslsrs_bg.wasm" } else { "" } }}
-	pnpm -C ./packages/csslsrs install
```

Comment:
I removed these steps (and the build-wasm below) because imo the commands should be stand alone, this also makes them faster in local where you have everything already, preventing unnecessary rebuilds / installs (it's easy to chain commands when you desire to)